src/prime_rl/trainer/train.py

```
@@ -281,7 +282,10 @@ def train(config: TrainerConfig):
 
         # Normalize all loss metrics globally before reporting
         for key, value in loss_metrics.items():
-            loss_metrics[key] = value / loss_scale
+            if key == "loss/entropy":
+                loss_metrics[key] = value / total_tokens
+            else:
+                loss_metrics[key] = value / loss_scale
```

argh this looks like code smell. seems like an easy footgun in the future. can we not move the normalization back in the grad accumulation loop? then the normalization factor is defined for each value explicitly which seems less likely to fail silently